components/project-card/project-card-detailed.jsx

```diff
@@ -122,13 +122,17 @@ class DetailedProjectCard extends React.Component {
   }
 
   renderVisitButton() {
+    let classes = classNames("btn btn-primary mb-3 mb-md-0 mr-md-3 d-flex justify-content-center align-items-center", {
```

Comment:
- move this to after `if (!this.props.contentUrl) return null;`
- fix indentation